src/placeos-log-backend.cr

```diff
@@ -6,8 +6,8 @@ require "./ext/log/broadcast_backend"
 module PlaceOS::LogBackend
   Log           = ::Log.for(self)
   STDOUT        = ActionController.default_backend
-  LOGSTASH_HOST = ENV["LOGSTASH_HOST"]?.presence
-  LOGSTASH_PORT = ENV["LOGSTASH_PORT"]?.try &.to_i?
+  UDP_LOG_HOST = ENV["UDP_LOG_HOST"]?.presence || ENV["LOGSTASH_HOST"]?.presence
+  UDP_LOG_PORT = (ENV["UDP_LOG_PORT"]?.presence || ENV["LOGSTASH_PORT"]?.presence).try &.to_i?
```

Comment:
This will silently keep using the old env vars, despite the deprecated `log_backend` method. May be worth providing a deprecation warning if only `LOGSTASH_*` vars are present so that these can be more safely removed following the same deprecation period as the old method.